packages/shared/locales/en.json

```diff
@@ -1571,6 +1571,23 @@
             "incoming": "Incoming",
             "outgoing": "Outgoing",
             "selfTransaction": "Self transaction"
+        },
+        "proposalType": {
+            "label": "Type",
+            "official": "Official",
+            "custom": "Custom"
+        },
+        "phase": {
+            "label": "Phase",
+            "upcoming": "Announcement",
+            "commencing": "Voting open",
+            "holding": "Counting starts",
+            "ended": "Counting stops"
```

Comment:
Is it possible to use the same locale's as we use for the labels?

Reply:
wrt translations for filters, the structure isnt that flexible...